migrations/1662381432402-test-transactions.js

```diff
@@ -0,0 +1,19 @@
+var db = require('./db')
+
+exports.up = function (next) {
+  const sql = [
+    `CREATE TABLE testing_addresses (
+        crypto_code TEXT NOT NULL,
+        address TEXT NOT NULL,
+        unique (crypto_code, address)
+      )`,
+    `ALTER TABLE cash_out_txs ADD COLUMN testing BOOLEAN NOT NULL DEFAULT FALSE`,
+    `ALTER TABLE cash_in_txs ADD COLUMN testing BOOLEAN NOT NULL DEFAULT FALSE`,
```

Comment:
Would prefer a more boolean-like name for this flag, since `testing` seems vague when reading the object as a whole. Something like `is_test_transaction` goes more directly straight to the meaning of this flag